lib/utils.pm

```diff
@@ -421,6 +422,27 @@ sub is_bridged_networking {
     return $ret;
 }
 
+=head2 set_hostname
+    set_hostname($hostname);
+
+Setting hostname according input parameter using hostnamectl.
+Calling 'reload-or-restart' to make sure that network stack will propogate
+hostname into DHCP/DNS
+
+if you change hostname using `hostnamectl set-hostname`, then `hostname -f` will fail with "hostname: Name or service not known"
+also DHCP/DNS don't know about the changed hostname, you need to send a new DHCP request to update dynamic DNS
+yast2-network module does "NetworkService.ReloadOrRestart if Stage.normal || !Linuxrc.usessh" if hostname is changed via `yast2 lan`
```

Comment:
break lines. Also, it looks like you are using markdown style and not POD. See https://perldoc.perl.org/perlpod.html#Formatting-Codes . Better use `C<hostnamectl set-hostname>` instead of something like

```
`hostnamectl set-hostname`
```